### PR TITLE
Create app

### PR DESCRIPTION
**Pull request by johnsonm325**

### CODE REVIEW

**Comment by johnsonm325** [comment]
@karelhala @Hyperkid123 I cloned this from the starter app and I'm trying to get the Sample Page to render at /insights/tasks/sample`, but it's not rendering. I'm not even able to get any console.log statements to print anywhere. I think I'm missing something simple, but any guidance here would help tremendously.

**Comment by Hyperkid123** [comment]
@johnsonm325 it will not render until you add entries to the cloud services config repo.

**Comment by adonispuente** [comment]
Im not sure if this is typical for starter apps, but currently notifications can break the application. To reproduce, 
- start up the app
- dispatch  2 alerts via the alert button
- clear out the notifications by pressing the x
- select SAP global filter (usually it happens here)
- press ' clear filters'
and youll be greeted with a bunch of notifications that slows the app significantly until it crashes

File: config/setupTests.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+            },
+          })
+        ),
+    },
+    isBeta: jest.fn(),
+    getUserPermissions: () =>
+      new Promise((res) =>
+        res([
+          {
+            permission: 'drift:*:*',
```
Comment (gabipodolnikova) [inline config/setupTests.js:32]: I know it is just a test set up, but this probably should not be drift here? Maybe for now just: `() => Promise.resolve()
    },`

Comment (johnsonm325) [reply config/setupTests.js:32]: good catch

**Review by adonispuente — APPROVED** [review]

**Pull request merged**